third_party/blink/renderer/core/layout/layout_box_hot.cc

```diff
@@ -102,16 +102,19 @@ const NGLayoutResult* LayoutBox::CachedLayoutResult(
   NOT_DESTROYED();
   *out_cache_status = NGLayoutCacheStatus::kNeedsLayout;
 
-  recordreplay::Assert("[RUN-1219-1440] LayoutBox::CachedLayoutResult Start %d %d %d",
-    (int) new_space.CacheSlot(), layout_results_.empty());
+  recordreplay::Assert("[RUN-1219-1657] LayoutBox::CachedLayoutResult Start %d %d %d %d",
+    this->RecordReplayId(),
+    (int) new_space.CacheSlot(),
+    layout_results_.empty()
+  );
 
   const bool use_layout_cache_slot =
       new_space.CacheSlot() == NGCacheSlot::kLayout && !layout_results_.empty();
   const NGLayoutResult* cached_layout_result =
       use_layout_cache_slot ? GetCachedLayoutResult(break_token)
                             : GetCachedMeasureResult();
 
-  recordreplay::Assert("[RUN-1219-1440] LayoutBox::CachedLayoutResult #1 %d %d %d %d",
+  recordreplay::Assert("[RUN-1219-1657] LayoutBox::CachedLayoutResult #1 %d %d %d %d",
```

Comment:
Changing the subissues in these assertions seems like busywork, can we avoid doing that?  If we find ourselves doing this too often I think it would be better to only have a single issue in the asserts.

Reply:
It was a simple search-replace.  The actual change is in the new assertions added, I just wanted to make things consistent.